src/Bridge/Doctrine/Purger/OrmPurger.php

```diff
@@ -79,9 +79,12 @@ public function create(PurgeMode $mode, PurgerInterface $purger = null): PurgerI
 
     /**
      * @inheritdoc
+     * @throws \Doctrine\DBAL\DBALException
      */
     public function purge()
     {
+        $this->manager->getConnection()->prepare('SET FOREIGN_KEY_CHECKS = 0;')->execute();
```

Comment:
I think the issue comes from the truncating of the tables not necessarily happening in a good order for cascade delete to work, and I guess some people may not want cascade delete for some reason, so I would be in favour of keeping this in.